Pages/FoodCustomer/Cart.cshtml

```diff
@@ -35,12 +50,57 @@
                     @Html.DisplayFor(modelItem => food.Food.Price)
                 </td>
                 <td>
-                    @Html.DisplayFor(modelItem => food.Quantity)
+                    <div class="number">
+                        <button type="button" class="minus">-</button>
+                        <input type="text" style="width: 50px; text-align: center;" value="@food.Quantity" />
+                        <button type="button" class="plus">+</button>
```

Comment:
If quantity exceeds the amount that a restaurant has. Will it be handled? (Not letting customer add any more of that food) Will the food page for customers show the quantity as 0 if someone reserves all of the supply?